README.md

```diff
@@ -32,6 +32,27 @@ The different alerting rules in this repository are:
 
 * [**Alert Rule QuotaExceeded**](https://gitlab.cee.redhat.com/rhtap/docs/sop/-/blob/main/o11y/alert-rule-QuotaExceeded.md)
 
+### SLO Alerts
+
+SLO (Service Level Objective) alert rules are rules defined to monitor and alert when a service or system is not meeting its specified service level objectives.
+
+#### **Usage Guidelines:**
```

Comment:
I'd suggest unbolding the headings. Having it everywhere will make the document busier.
```suggestion
#### Usage Guidelines:
```

Reply:
Done